charts/harvester-cloud-provider/harvester-cloud-provider/0.1.300+up0.1.3/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
Can we also restructure these commits two 3 commits?

Commit 1 (package.yaml), Commit 3 (package.yaml) and Commit 2+4 (just rename this commit `make charts`).